script.js

```diff
@@ -249,15 +278,61 @@ function handleLocalitiesLayer(geojson) {
   });
 
   // Add data to the map
-  localitiesLayer.addTo(map);
+  citiesLayer.addTo(map);
 
-  // Move the map view so that the localitiesLayer is visible
-  map.fitBounds(localitiesLayer.getBounds(), {
+  // Move the map view so that the citiesLayer is visible
+  map.fitBounds(citiesLayer.getBounds(), {
     paddingTopLeft: [12, 120],
     paddingBottomRight: [12, 12]
   });
 
-  return localitiesLayer;
+  return citiesLayer;
+}
+
+function handleCountiesLayer(geojson) {
+  const layerOptions = {
+    style: feature => {
+      // style states based on whether their moratorium has passed
+      if (feature.properties.passed === true) {
+        return {
+          color: "#4dac26",
+          fillColor: "#b8e186",
+          fillOpacity: fillOpacity,
+          weight: strokeWeight
+        };
+      } else if (feature.properties.passed === false) {
```

Comment:
@timhitchins you can also simplify this by doing:

```js
} else if (!feature.properties.passed) {
```

Reply:
I changed the CARTO queries in f8bd200 to avoid client side changes to the properites.